### ISSUE TITLE
Adding a new connection crashes the page as you type into the host field on Chrome

### ISSUE THREAD

**Issue by taylorlapeyre** [taskforcesh/issues#148]

**Comment by manast** [comment]
I cannot reproduce this issue. Are you it was not some temporal glitch in Chrome? the connection modal has not been modified in months and it seems to be working for everybody else...

**Comment by taylorlapeyre** [comment]
I'm using Arc. I was eventually able to enter it, but the page consistently froze and eventually crashed every time I tried to paste in a connection URL, or after typing a lot of characters in succession. I don't think it was a browser bug!

**Comment by manast** [comment]
It may be an issue specific of the Arc browser. Nothing I can reproduce with Chrome or Safari no matter how many characters or how fast I paste or type them in. Nobody has complained about this before, however Arc doesn't seem to be a very popular browser so I guess that is the issue.

**Comment by wernermorgenstern** [comment]
I had the issue in Chrome today.
When I copy and paste the Redis Cloud Connection String into the `host` field (e.g. `redis-85737.AAA.us-REGION-2-1.ec2.cloud.rlrcp.com:85737`)
(Random port, and randomized), Chrome got stuck, and even crashed.

However, if I copy and paste the string as `redis-85737.AAA.us-REGION-2-1.ec2.cloud.rlrcp.com` (without the port number at the end), then it worked.

So I am not sure if Taskforce.sh validates the Host string as you type it in or copy and paste? And when the `:` with port number at the end, it is invalid, and Chrome crashes?

**Comment by manast** [comment]
I will try to reproduce.

**Comment by manast** [comment]
For me it works normally:
![image](https://github.com/taskforcesh/issues/assets/95200/58ad0351-fedf-4e2a-a658-53743e51c3d8)

**Comment by manast** [comment]
After pasting the string, what do you do? I just click outside of the field and I get the error that the string must be a domain or IP address.

**Comment by wernermorgenstern** [comment]
For me to get it to work, I had to close the browser or the tab in Chrome, and then I just copied and pasted the URL without the port number.
And then I was able to do the rest.

I just tried to do it again, and now it works.  With the string I gave you.

When I use the real Redis Connection string, then it hangs.

Does the Website use a lookup, to make sure the URL is reachable?
I didn't see anything in Chrome Debugger.

**Comment by manast** [comment]
No, it does not do anything like that, it is just a regular expression, standard angular code.
